### DISCUSSION TITLE
counter by line crossing

### DISCUSSION THREAD

**Discussion by Rasantis** [roboflow/supervision#755] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Hello, would you have any sample .py script for people counting that can be run in VSCode? I have been trying to adapt the scripts from Google Colab, but it's been difficult and I'm not succeeding. Could you assist me? Happy 2024 to the entire Supervision team, and thank you very much for making this incredible product.

**Comment by SkalskiP** [comment]
Hi, @Rasantis 👋🏻 ! Thanks for your interest in `supervision`! Would you be interested in [this](https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/how-to-detect-and-count-objects-in-polygon-zone.ipynb) Colab, but in script format?

**Comment by SkalskiP** [comment]
I'm converting this issue into a discussion and moving it into a Q&A section.
